### PR TITLE
fix: add functionConfigurationsLocation config

### PR DESCRIPTION
**Pull request by generalman025**
I would like to add functionConfigurationsLocation config to determine a path of functionConfiguration files

### CODE REVIEW

File: src/getAppSyncConfig.js [hunk 1]
```diff
@@ -13,6 +13,11 @@ const directLambdaMappingTemplates = {
   response: directLambdaResponse,
 };
 
+const mappingTemplateType = {
+  mappingTemplate: 'mappingTemplate',
+  functionConfiguration: 'functionConfiguration',
+};
+
```
Comment (bboure) [inline src/getAppSyncConfig.js:20]: It all LGTM!
Is there any added value to this "mapping"?
We already use hardcoded strings for `request` and `response` for `type`

possibly, we could use constants everywhere if we wanted to improve it slightly.

**Review by bboure — COMMENTED** [review]
Thank you @generalman025 for adding support for that! 🙏 

I added one small question/comment but it LGTM

File: src/getAppSyncConfig.js (hunk 1 above)
Comment (generalman025) [reply src/getAppSyncConfig.js]: Thanks for your questions @bboure. I think we should store this "mapping" to another location and refactor some hardcoded strings into it as I recently added.

**Review by bboure — APPROVED** [review]
LGTM 
Thank you!

**Pull request merged**